packages/webview_flutter/webview_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.5.1
 
 * Updates minimum supported SDK version to Flutter 3.7/Dart 2.19.
+* Add support to show JavaScript dialog. See `PlatformWebViewController.setJavaScriptAlertDialogCallback`.
```

Comment:
nit:
```suggestion
* Adds support to show JavaScript dialog. See `PlatformWebViewController.setJavaScriptAlertDialogCallback`.
```

This can also list the other two callback methods.